src/main/java/edu/umich/lib/dor/replicaexperiment/service/ReplicaService.java

```diff
@@ -20,6 +20,8 @@ public Replica createReplica(InfoPackage infoPackage, Repository repository) {
         var replica = new Replica();
         replica.setInfoPackage(infoPackage);
         replica.setRepository(repository);
+        infoPackage.addReplica(replica);
+        repository.addReplica(replica);
```

Comment:
Moving these calls here seems to also solve the bi-directional data problem that causes the integration tests to fail. I think this is a minor improvement and certainly reduces the code in `Deposit.execute` and `Replication.execute`, so I'm going with it.